sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/BlockBlobsImpl.java

```diff
@@ -196,11 +199,15 @@ public BlockBlobsImpl(AzureBlobStorageImpl client) {
         if (cpkInfo != null) {
             encryptionAlgorithm = cpkInfo.getEncryptionAlgorithm();
         }
+        String encryptionScope1 = null;
+        if (encryptionScope != null) {
+            encryptionScope1 = encryptionScope.getEncryptionScope();
+        }
```

Comment:
@JonathanGiles, this code was generated with the latest version of the Java generator.  It might be best to bring these issues up with @jianghaolu.  I think issues with generated code are outside the scope of this PR.

Reply:
Got it - I'll redirect this comment to @jianghaolu then. Thanks.